src/actions/task-definitions.js

```diff
@@ -89,6 +89,13 @@ tasks.moveCellDown = new UserTask({
 })
 
 
+tasks.pinCell = new UserTask({
+  title: 'Pin this cell',
```

Comment:
maybe change the title to "pin this editor"

Reply:
I think "pin this editor" could be confusing too, tbh, since all the cells in sum are the editor. "pin this cell editor" is probably what we want.